store/src/db.rs

```diff
@@ -23,10 +23,7 @@ impl Db {
 
         let conn = pool.get().await?;
 
-        info!(
-            sqlite = format!("{}", config.sqlite.display()),
-            "Connected to the DB",
-        );
+        info!(sqlite = format!("{}", config.sqlite.display()), "Connected to the DB",);
```

Comment:
Surprising that rustfmt/clippy didn't remove the trailing comma here.

Reply:
I guess that is because it is an unstable feature:

https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#trailing_comma

There are quite a few configuration options that i'd like to use, but are on nightly only :(. For this one I edited it manually.